### PR TITLE
France launch

### PR DESCRIPTION
**Pull request by adeclosset**
Most site (except legals) now in French.

@angusbayley @tpillairavin can you check if the ids used for tracking in the header (in include/fr) files are correct? They are referring to English elements (e.g login) that are different in France (in that example it's contact us). Same for "produit" and "tarifs".

@harrison can you check if all is good to deploy?

### CODE REVIEW

**Comment by harrison** [comment]
@jamesshedden do you want to review?

**Comment by jamesshedden** [comment]
@harrison @adeclosset Hey yeah can take this, will do shortly

File: app/pages/fr/a-propos/index.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      </div>
+    </div>
+  </div>
+
+  <div class="sticky-nav">
+    <div ng-gc-sticky-nav class="sticky-nav__inner u-background-light-gray u-text-heading u-text-xxs u-text-light u-text-no-smoothing">
+      <div class="site-container u-padding-Vm">
+        <ul class="u-pull-start u-margin-Txxs">
+          <li class="sticky-nav__item">
+            <a href="#about-us" du-scrollspy ng-gc-smooth-scroll class="sticky-nav__link">
```
Comment (jamesshedden) [inline app/pages/fr/a-propos/index.html]: Change `href` to match link text eg `href="a-propos"`

File: app/pages/fr/a-propos/index.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+              À propos
+            </a>
+          </li>
+          <li class="sticky-nav__item">
+            <a href="#team" du-scrollspy ng-gc-smooth-scroll class="sticky-nav__link">
```
Comment (jamesshedden) [inline app/pages/fr/a-propos/index.html]: Same - ``href="notre-equipe"``

File: app/pages/fr/a-propos/index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+              Notre équipe
+            </a>
+          </li>
+        </ul>
+        <!--<a href="https://gocardless.com/blog" class="sticky-nav__cta btn btn--small btn--hollow u-pull-end u-text-transform-none u-text-light u-text-xxs u-text-no-smoothing">
```
Comment (jamesshedden) [inline app/pages/fr/a-propos/index.html]: Can delete this commented-out code if we're not using it for now - can always retrieve from GitHub if we need it back at any point

File: app/pages/fr/a-propos/index.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+          Visitez notre blog
+        </a>-->
+      </div>
+      <hr class="u-margin-An">
+    </div>
+  </div>
+
+  <div class="section-scroll-target" id="about-us">
```
Comment (jamesshedden) [inline app/pages/fr/a-propos/index.html]: When we change the link `href` we'll need to change the `id` on these - `id="a-propos"`

File: app/pages/fr/a-propos/index.html [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+          </div>
+        </div>
+      </div>
+    </div>
+
+  </div>
+
+  <hr class="u-margin-An">
+
+  <div class="section-scroll-target" id="team">
```
Comment (jamesshedden) [inline app/pages/fr/a-propos/index.html]: `id="notre-equipe"`

File: app/pages/fr/faq/commercant/index.html [hunk 6]
```diff
@@ -0,0 +1,547 @@
...
+      </h1>
+    </div>
+  </div>
+
+  <div class="site-container u-padding-Vxxl grid">
+    <div class="u-padding-Vxl">
+      <div class="grid__cell u-size-4of12">
+        <div class="u-size-5of6">
+          <ul class="nav nav-tabs">
+            <li><a href="#overview-tab" class="nav-tabs__link u-block u-padding-Bs u-text-light u-text-no-smoothing"
```
Comment (jamesshedden) [inline app/pages/fr/faq/commercant/index.html]: `href="en-bref"` - etc same for all links in this `nav-tabs` component, be good if they reflect the text in the link as they're used in the URLs

File: app/pages/fr/faq/commercant/index.html [hunk 7]
```diff
@@ -0,0 +1,547 @@
...
+            </a></li>
+          </ul>
+          <hr class="u-size-11of12">
+          <p class="para u-padding-Txs">Une question? Appelez nous au +33 (0) 9 75 18 42 95</p>
+        </div>
+      </div>
+
+      <div class="grid__cell u-size-8of12">
+
+        <article id="overview-tab" ng-gc-tabby-content>
```
Comment (jamesshedden) [inline app/pages/fr/faq/commercant/index.html]: Then need to make sure the `id` matches in these `<article>` elements eg `id="en-bref"`

File: app/pages/fr/tarifs/index.html [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+
+  <div class="page-hero--pricing">
+    <div class="site-container">
+      <div class="site-header site-header--invert u-relative u-cf">
+        {% include "fr/_site_header_invert.html" %}
+      </div>
+
+      <div class="grid pricing-options u-center u-padding-Bxl">
+
+        <div class="grid__cell u-padding-Vxl u-padding-Rxs">
```
Comment (jamesshedden) [inline app/pages/fr/tarifs/index.html]: Think this `<div>` needs a smaller width, so be good to add an extra class `u-size-1of2`, so:

`<div class="grid__cell u-padding-Vxl u-padding-Rxs">`

would become:

`<div class="grid__cell u-size-1of2 u-padding-Vxl u-padding-Rxs">`

**Comment by jamesshedden** [comment]
@adeclosset Hey, awesome work - few comments, grab me if anything doesn't make sense or with any other problems/questions! :+1:

**Comment by angusbayley** [comment]
@jamesshedden I don't mind about the GA enhanced link attribution id's being in the wrong language (nor does GA), but if it amounts to unacceptably messy code just ping me and I will set 30 mins aside to tidy up

**Comment by jamesshedden** [comment]
@angusbayley Hey no probs with the tracking IDs being in different languages - it's just the IDs we're using on elements that affect the URLs that we should have in the correct language.

**Comment by angusbayley** [comment]
@jamesshedden not sure what you mean. If in the /fr site there is `id="track-nav-pricing" href="/tarifs"` is that ok or not ok?

**Comment by octaveauger** [comment]
@harrison @jamesshedden changes done - ready to go?

**Comment by jamesshedden** [comment]
@octaveauger Nice one, all good :+1:

**Pull request merged**